doc/source/dev/releasing.rst

```diff
@@ -111,18 +112,30 @@ Here is a complete list of artifacts created for a release:
 
 Source archives, Changelog and README are built by running ``paver release`` in
 the repo root, and end up in ``REPO_ROOT/release/``.  Do this after you've
-created the signed tag locally.  If this completes without issues, push the release
-commit (not the tag, see section above) to the scipy repo.
+created the signed tag locally. ``paver release`` will be sensitive to the version
+of Cython available in your build environment, so make sure your version matches the
+minimum requirements for the release. If this completes without issues, push the release
+commit (not the tag, see section above) to the scipy repo. If ``pavement.py`` is causing
+issues, it is also possible to simply use ``python setup.py sdist`` and perform the
+release notes task from ``pavement.py`` by hand.
 
 To build wheels, push a commit to the master branch of
-https://github.com/MacPython/scipy-wheels .  This triggers builds for all needed
+https://github.com/MacPython/scipy-wheels . This triggers builds for all needed
```

Comment:
Shouldn't the push be in the branch used for the release rather than the master branch? The master branch also uploads wheels to a different location than the branches.

Reply:
Yeah, I found that confusing initially too. Reading through it again I thought maybe they were referring to just a simple trigger of the wheels master branch first to make sure everything is ok (which indeed it was not for SciPy recently) there before branching on the wheels repo.

This may be more import for SciPy which has less frequent cron jobs maybe.

Reply:
This is indeed outdated - we used to have `master` for releases and `dev` for the latest changes, but we changed that to the more intuitive `master` for latest and `1.y.z` for release branches (IIRC, didn't double check).